tests/acceptance/cli_test.py

```diff
@@ -552,24 +570,15 @@ def test_config_compatibility_mode(self):
         self.base_dir = 'tests/fixtures/compatibility-mode'
         result = self.dispatch(['--compatibility', 'config'])
 
-        assert yaml.load(result.stdout) == {
-            'version': '2.3',
-            'volumes': {'foo': {'driver': 'default'}},
-            'networks': {'bar': {}},
-            'services': {
-                'foo': {
-                    'command': '/bin/true',
-                    'image': 'alpine:3.10.1',
-                    'scale': 3,
-                    'restart': 'always:7',
-                    'mem_limit': '300M',
-                    'mem_reservation': '100M',
-                    'cpus': 0.7,
-                    'volumes': ['foo:/bar:rw'],
-                    'networks': {'bar': None},
-                }
-            },
-        }
+        assert yaml.load(result.stdout) == COMPOSE_COMPATIBILITY_DICT
+
+    @mock.patch.dict(os.environ)
+    def test_config_compatibility_mode_from_env(self):
+        self.base_dir = 'tests/fixtures/compatibility-mode'
+        result = self.dispatch(['--compatibility', 'config'])
```

Comment:
I might misunderstand the testcase, but shouldn't you set env variable then run `config` command _without_ `--compatibility` ?

Reply:
Whoops and yes it's what I intended to do but I forgot to do it fully 🤦‍♂️
I'll fix the test as soon as i get a real computer in front of me.

Reply:
:heavy_check_mark: This should be fixed with commit 85730b3e9e4595488caf4c878f66a5dd80404af9